test/charts/charts.sh

```diff
@@ -49,29 +50,5 @@ for chart in $pwd/charts/*; do
     fi
     cd ..
     echo "Done."
-
```

Comment:
I think we should also be able to remove the allowed apiversions at the top of the file now

Reply:
done